Common.mk

```diff
@@ -163,7 +163,7 @@ HELM_SOURCE_REPOSITORY?=$(REPO)
 HELM_SOURCE_IMAGE_REPO?=$(IMAGE_REPO)
 HELM_GIT_TAG?=$(GIT_TAG)
 HELM_TAG?=$(IMAGE_TAG)
-HELM_DIRECTORY?=.
+HELM_DIRECTORY?=$(PROJECT_ROOT)
```

Comment:
HELM_DIRECTORY now defaults to project root.

PROJECT_ROOT is release branch aware, so HELM_DIRECTORY is not as well.

This was necessary to get helm working in release branch directories.